planner_cspace/test/src/test_grid_astar.cpp

```diff
@@ -91,6 +95,70 @@ TEST(GridAstar, ParallelSearch)
   }
 }
 
+TEST(GridAstar, SearchWithMultipleStarts)
+{
+  using Vec = CyclicVecInt<1, 1>;
+  GridAstar<1, 1> as(Vec(16));
+  as.setSearchTaskNum(1);
+
+  const auto cb_cost = [](
+      const Vec&, const Vec&, const std::vector<GridAstar<1, 1>::VecWithCost>&, const Vec&) -> float
+  {
+    return 1.0;
+  };
+  const auto cb_cost_estim = [](const Vec&, const Vec&) -> float
+  {
+    return 1e-6;
+  };
+  // create search table of graph edges (relative vector to the connected grid)
+  // 0: connected to 2-14
+  // 1: connected to 2-14
+  // 2-14: connected to 2-15
+  std::vector<Vec> search[16];
+  for (int i = 2; i < 15; ++i)
+  {
+    search[0].push_back(Vec(i));
+    search[1].push_back(Vec(i - 1));
+    for (int j = 2; j < 15; ++j)
```

Comment:
The condition should be j <= 15, and L128 can be removed.

Reply:
it intended to show the shape of 
```
       15
    /   |   \
  /     |     \
 2 ... 10 ... 14
  \     |     /
    \   |   /
        0
```
but okay to be simplified.